tests/teal/router/questionableFP_approval_v8.teal

```diff
@@ -0,0 +1,512 @@
+#pragma version 8
```

Comment:
The next 8 `teal` files, previously were inlined strings in `pyteal/compiler/compiler_test.py::test_router_app`

@ahangsu

Reply:
thx, the compiler_test.py diff is hard to digest somehow.

Reply:
Yeah, I know. But I did manually copy and paste the strings that were there into the new files.

Reply:
yes, that makes things much easier to follow :)